src/Gossip.csproj

```diff
@@ -1,6 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>0.1.0</Version>
+    <Authors>Wayfair</Authors>
+    <Description>Extends the functionality of Dapper and allows performing database queries in .NET easily and quickly</Description>
+    <Copyright>Wayfair ©2021</Copyright>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>

```

Comment:
Looking at the build output [here](https://github.com/wayfair-incubator/Gossip/runs/4485852004?check_suite_focus=true#step:5:10), it looks like it is unable to find this license file. You may need to include a similar section as what we've defined for WaySON [here](https://github.com/wayfair-incubator/wayfair-text-json/blob/main/src/WaySON/WaySON.csproj#L28-L31), although with a different path since this project has everything sitting in `src` and not `src/Gossip`.

Reply:
Just to add on and fix my previous comment, it'd be this `..\LICENSE"` in your path instead of `..\..\LICENSE` in your include.